
    console.log("this is in the catch " + message);